### PR TITLE
Update NEWS for 1.4.2 release

### PR DESCRIPTION
**Pull request by samj1912**

### CODE REVIEW

File: NEWS.txt [hunk 1]
```diff
@@ -1,3 +1,14 @@
+Version 1.4.2 - 2017-05-08
+** Bug
+    * [PICARD-969] - Search dialog webservices get queued behind matched album requests
```
Comment (Sophist-UK) [inline NEWS.txt]: Not sure whether this is a bug or enhancement.

File: NEWS.txt [hunk 2]
```diff
@@ -1,3 +1,14 @@
...
+    * [PICARD-1034] - Picard not seeing TOPE and TOAL
```
Comment (Sophist-UK) [inline NEWS.txt]: Ditto

File: NEWS.txt [hunk 3]
```diff
@@ -1,3 +1,14 @@
...
+    * [PICARD-1053] - Picard does not stop analyzer while moving
+    * [PICARD-1055] - picard hangs with: RuntimeError: maximum recursion depth exceeded in cmp
+    * [PICARD-1077] - ID3v2.4 text encoding settings are not saved correctly
+    
+** Improvement
+    * [PICARD-1070] - The "Convert Unicode punctuation characters to ASCII" function only works in certain tags
```
Comment (Sophist-UK) [inline NEWS.txt]: Opposite - this feels more like a bug.

Comment (samj1912) [inline NEWS.txt]: These are all directly generated from jira. You can update the issues on jira. They were all filed/fixed by you iirc.

Comment (Sophist-UK) [inline NEWS.txt]: Yes - I have changed the tickets if you want to regenerate this.

**Pull request merged**